SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -20,37 +20,35 @@ void SpatialEventTracer::TraceCallback(void* UserData, const Trace_Item* Item)
 		return;
 	}
 
-	uint32_t ItemSize = Trace_GetSerializedItemSize(Item);
 	// Depends whether we are using rotating logs or single-log mode (where we track max size).
 	const bool bTrackFileSize = EventTracer->MaxFileSize != 0;
-	if (!bTrackFileSize || (EventTracer->BytesWrittenToStream + ItemSize <= EventTracer->MaxFileSize))
+	if (!bTrackFileSize || (EventTracer->BytesWrittenToStream < EventTracer->MaxFileSize))
```

Comment:
Ah. Apologies. Missed this breaking change off the release notes. Changes look good though.

Reply:
No worries! Thanks :)